site/plugins/sidebar/sidebar.py

```diff
@@ -169,10 +169,7 @@ def _prepare_sidebar(self, destination, lang, template):
 
         month_list = self._build_month_post_list(lang)
         context['month_list'] = month_list
-        deps_dict['month_list'] = []
-
-
-        deps_dict['month_list'] += [(month.year + '/' + month.month, month.friendly_name) for month in month_list]
+        deps_dict['month_list'] = [(month.year + '/' + month.month, month.friendly_name) for month in month_list]
```

Comment:
Why this change? What has to do with the tag cloud?

Reply:
Mi punto es que este cambio ya está en master, entonces no entiendo qué estamos viendo.

Ah, ahora veo que el Pull Request que armaste es raro: `
marian-vignau wants to merge 7 commits into i4_rss_feed from i5_tag_cloud `, ¿no deberías estar haciendo un PR contra master?